### ISSUE TITLE
Brand-new planet cannot upgrade to newest OTA

### ISSUE THREAD

**Issue by hosted-fornet** [urbit/urbit#5301]
Started a brand-new planet after the app distribution OTA had been released. Cannot upgrade to most recent OTA. 

Result of
```
|ota (sein:title our now our) %kids
```
https://pastebin.com/kGyNH5cE

Result of
```
|merge %kids ~litdev %kids, =gem %only-that
|merge %home our %kids, =gem %only-that
+trouble
```
https://pastebin.com/n7Msnn06

Expected behavior: Brand-new planet should upgrade to most recent version without any problems.

**Comment by philipcmonk** [comment]
Very strange error, it's as though your version of the %mime mark is several months old.  What version of urbit did you use to start the planet?  If it was pre-v1.5 it would have had an old pill.

**Comment by philipcmonk** [comment]
Was this on port?

**Comment by hosted-fornet** [comment]
This was from command line. Looking at my bash history, it looks like I did the initial boot with v1.2 binaries, then started using v1.5 from that point on. I don''t recall for sure whether I started OTAs before or after switching to v1.5, but I think after.. Could this have broken things?

**Comment by philipcmonk** [comment]
Yes, the v1.2 pill was quite old, probably between v2.50 and v2.75.  You can probably upgrade it so you don't have to breach, but it could take a bit of time.  The basic strategy is to take smaller steps -- ie instead of trying to update straight to v2.102, go to like v2.60 then v2.80 then v2.100 then v2.102.  Will take some experimentation to see what sequence works, though.

You can try to install an older version (75 in this case, which corresponds to i think v2.74) with:

```
|merge %home ~litdev %kids, =gem %only-that, =cas ud+75
```

The most suspicious updates were around v2.75 and v.2.100, but I'd recommend a simple binary search, starting around 75.  If that works go to 87 then 100 then 101 then 102 or whatever it lets you do.  If 75 fails back up five at a time until you find one that works.

**Comment by hosted-fornet** [comment]
Awesome, thanks for the assistance. I'll try that out and post a follow-up here, but I think this issue is resolved.
